plugins/module_utils/wdc_redfish_utils.py

```diff
@@ -0,0 +1,417 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import datetime
+import re
+import time
+import tarfile
+import socket
+
+from ansible.module_utils.urls import fetch_file, open_url
+from ansible_collections.community.general.plugins.module_utils.redfish_utils import RedfishUtils
+
+from ansible.module_utils.six.moves.urllib.parse import urlparse, urlunparse
+
+try:
+    import ipaddress
+    IPADDRESS_AVAILABLE = True
+except ImportError:
+    IPADDRESS_AVAILABLE = False
```

Comment:
Is this dependency still needed?

Reply:
Removed [7d838381d41f0620ea3b7406cda0762e971e8143](https://github.com/ansible-collections/community.general/pull/4885/commits/7d838381d41f0620ea3b7406cda0762e971e8143)